src/access_nri_intake/cli.py

```diff
@@ -141,6 +145,16 @@ def build():
         ),
     )
 
+    parser.add_argument(
+        "--data_base_path",
+        type=str,
+        default="",
```

Comment:
Does it make more sense to use a base path of `'./'` rather than an empty default? I've tried to follow through the use of data_base_path & I really can't tell - `'./'` intuitively feels cleaner to me, but I feel like there might be a good reason why you haven't done that here I'm not seeing?

Reply:
I chose it because it no-ops `os.path.join`:

```ipython
In [2]: os.path.join("", "/my/existing/path")
Out[2]: '/my/existing/path'
```

`./` does do the same, however:

```ipython
In [3]: os.path.join("./", "/my/existing/path")
Out[3]: '/my/existing/path'
```

However, the output does differ if the path you join to is relative (although only syntactically, the effective output is identical):

```ipython
In [4]: os.path.join("./", "my/existing/path")
Out[4]: './my/existing/path'

In [5]: os.path.join("", "my/existing/path")
Out[5]: 'my/existing/path'
```

Reply:
Cool - I've done a bit of looking through the docs & it looks like functionally they're equivalent in every way - if we prepend a relative path to an absolute path, the relative path will be ignored.

I think it would make more sense to use `"./"` just for consistency with `--build_base_path` & `--catalog_base_path`, but I don't think it's crucial.